.env

```diff
@@ -5,7 +5,7 @@
 # - The codebase has to stay open source, whether it was modified or not
 # - You can not repackage or sell the codebase
 # - Acquire a commercial license to remove these terms by emailing: license@cal.com
-NEXT_PUBLIC_LICENSE_CONSENT=
+NEXT_PUBLIC_LICENSE_CONSENT='agree'
```

Comment:
correct. we keep this empty, however we will get rid of this .env variable once we have the proper onboarding wizard where you accept the license @zomars

Reply:
I don't think we can modify the env variable on runtime (wizard) but we can come up with another way to accept it maybe.

Reply:
Yea we can't really lose the .env altogether for runtime config, but there are probably other ways to handle the consent part specifically

Reply:
@zomars we dont need the license check in the .env at all. its just a reminder how the license works. it just needs to be a check in the onboarding wizard and saved in the database